README.md

```diff
@@ -224,6 +224,10 @@ function onError(error){
   alert("Sorry! Cannot view document.");
 }
 ```
+### Close a Document File ###
+```js
+cordova.plugins.SitewaertsDocumentViewer.viewDocument()
```

Comment:
This feature is not supported on all platforms. Should have a windows specific doc.